app/move/controllers/update/base.test.js

```diff
@@ -607,6 +609,10 @@ describe('Move controllers', function() {
         expect(moveService.update).to.not.be.called
       })
 
+      it('should set the confirmation message', async function() {
```

Comment:
Copy paste jobby? Guessing should be `should not set...`?

Reply:
Corrected